tools/cli/admin.go

```diff
@@ -1029,6 +1029,15 @@ func newAdminFailoverCommands() []cli.Command {
 					Usage: "Optional domains to failover, eg d1,d2..,dn. " +
 						"Only provided domains in source cluster will be failover.",
 				},
+				cli.IntFlag{
+					Name: FlagFailoverDrillWaitTimeWithAlias,
```

Comment:
Is this a required flag when cron schedule is specified?

Reply:
Yes. Will update it